src/ui/public/index_patterns/_calculate_indices.js

```diff
@@ -17,10 +17,10 @@ define(function (require) {
     function compileOptions(pattern, timeFieldName, start, stop) {
       const constraints = {};
       if (start) {
-        constraints.min_value = { gte: moment(start).valueOf() };
+        constraints.max_value = { gte: moment(start).valueOf() };
```

Comment:
I'm suspicious of just passing a date to `moment()` here. If start is passed as a string we shouldn't be using moment's default format guessing to convert it to a timestamp.